tcf_website/views/review.py

```diff
@@ -115,9 +115,8 @@ def edit_review(request, review_id):
         form = ReviewForm(request.POST, instance=review)
         if form.is_valid():
             form.save()
-            messages.success(
-                request,
-                f'Successfully updated your review for {form.instance.course}!')
+            messages.success(request,
+                             f'Successfully updated your review for {form.instance.course}!')
```

Comment:
I believe this is what it originally was.

Reply:
I'm not sure if it matters (because it doesn't really hurt readability in either case) but technically, you'll see that there is a diff before and after this PR:
![image](https://user-images.githubusercontent.com/7676354/106652828-ffa36c80-6563-11eb-95a5-089f0f20f6cb.png)
which you can check with either `git diff` or GitHub web UI.

Reply:
Ah you're right, I thought it reverted to what it originally was but clearly not. Should be okay in this case?